rhc-ose-ansible/roles/registry/tasks/auth_layer_nginx.yaml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Install NGinx Repository
```

Comment:
nit picking: if you have another change to make, make the upper/lower case of NGinx consistent with the rest of the implementation.